schema.d.ts

```diff
@@ -386,6 +387,34 @@ export interface TypeReference extends Reference {
   end?: number;
 }
 
+/**
+ * An interface that describes the properties and methods of an object.
+ */
+export interface InterfaceDeclaration {
+  kind: 'interface';
+
+  name: string;
+
+  /**
+   * A markdown summary suitable for display in a listing.
+   */
+  summary?: string;
+
+  /**
+   * A markdown description of the class.
+   */
+  description?: string;
+
+  /**
+   * The interfaces that this interface extends.
+   */
+  supertypes?: Array<Reference>;
+
+  members?: Array<ClassMember>;
+
+  source?: SourceReference;
```

Comment:
fwiw I still dont know what a SourceReference really is or how its supposed to be documented

I think the PR for SourceReferences was merged by you and Alexander, I added some comments after merging (because I had missed the PR), but I still dont really know. Would be nice to have some clarification